tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -89,6 +90,15 @@
 autograph = lazy_loader.LazyLoader(
     "autograph", globals(),
     "tensorflow.python.autograph.impl.api")
+interleave_ops = lazy_loader.LazyLoader(
+    "interleave_ops", globals(),
+    "tensorflow.python.data.experimental.ops.interleave_ops")
+scan_ops = lazy_loader.LazyLoader(
+    "scan_ops", globals(),
+    "tensorflow.python.data.experimental.ops.scan_ops")
+random_ops = lazy_loader.LazyLoader(
+    "random_ops", globals(),
+    "tensorflow.python.ops.random_ops")
```

Comment:
I understand why we need to load `interleave_ops` and `scan_ops` lazily but why do we need `random_ops`?

On a related note, it makes sense to delay graduating this symbol until we graduate `scan` and `sample_from_datasets` (as it is less than ideal to have non-experimental API that depends on experimental API for its implementation).

Reply:
sure, will look into scan_ops before this. However, as per your comment in the doc `sample_from_datasets` is not ready yet.